keyboards/pimoroni/keybow2040/config.h

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+#define RGB_MATRIX_LED_COUNT (4 * 4)
+#define RGB_MATRIX_KEYPRESSES
+
+/*
+ * LED driver chip count: one IS31FL3731
+ */
+#define DRIVER_COUNT 1
+
```

Comment:
Dunno why this change is needed?

Reply:
All three of these defines are no longer necessary.

Reply:
Because they are no longer required.

Reply:
OK, I guess the DRIVER_COUNT is obsolete since 84df69572ce47f74213869120211f9db371cd3cf

And RGB_MATRIX_LED_COUNT is obsolete since 95681b8ff4a92aacd0249e124d34cf16e510175e

And DRIVER_COUNT since 84df69572ce47f74213869120211f9db371cd3cf

Dunno what the issue is with RGB_MATRIX_KEYPRESSES?

Reply:
It's implied when you have reactive effects enabled.